### PR TITLE
Refactor hard-coded strings to use string_ids.h constants

### PR DESCRIPTION
**Pull request by octaroot**
This is a little step towards a bit more tidy codebase.

Jesus this was tedious work. I manually added a bunch of strings to the string_ids.h file and used the constants instead of hardcoded ids. I also added one sprite constant (floppy disk icon) and fixed a couple hardcoded sprite ids.

### CODE REVIEW

File: src/localisation/string_ids.h [hunk 1]
```diff
@@ -1730,10 +1764,14 @@ enum {
 	STR_SCROLL_DOWN_FAST_TIP = STR_SCROLL_LEFT_TIP + 8,
 	STR_SCROLL_UP_DOWN_TIP = STR_SCROLL_LEFT_TIP + 9,
 
+	STR_MAP = 3157,
+	STR_GRAPH = 3158,
 	STR_LIST = 3159,
 
 	STR_SELECT_NUMBER_OF_CIRCUITS_TIP = 3160,
 
+	STR_EMPTY_STRING = 3165,
```
Comment (IntelOrca) [inline src/localisation/string_ids.h]: Could you rename this to place holder as that is what its typically used for?

**Comment by IntelOrca** [comment]
Yep, thanks for doing this though.

File: src/localisation/string_ids.h (hunk 1 above)
Comment (octaroot) [inline src/localisation/string_ids.h]: Sure, done.

**Comment by octaroot** [comment]
Anything I can do to help :)

**Pull request merged**